pkg/fanal/secret/scanner.go

```diff
@@ -409,6 +412,7 @@ func (s *Scanner) Scan(args ScanArgs) types.Secret {
 		return findings[i].Match < findings[j].Match
 	})
 
+	log.Logger.Debugf("%q contains secrets", args.FilePath)
```

Comment:
Do we need it? The detected secrets are shown in the result.

Reply:
yes, sure. It's a redundant message